Islandora-Devops/ansible-role-fits
# Adds microservice to connect Islandora to FITS webservice

**GitHub Issue**: (link)

* Other Relevant Links (Google Groups discussion, related pull requests,
 Release pull requests, etc.)

# What does this Pull Request do?

Downloads, configures and installs a microservice to connect to FITs webservice.  The connectors have been added already to the Alpaca role.

# What's new?
All new code.
It will create a new media type dynamically - each new field type coming back from the fits xml will be added to the media type on ingest.

# How should this be tested?
Once this role is run you should be able to add any Original File and a new Fits Metadata media associated with the owning object.


# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
@Islandora-Devops/committers
